packages/dotlottie-js/src/common/lottie-animation-common.ts

```diff
@@ -34,10 +34,14 @@ export class LottieAnimationCommon {
 
   private _playMode: PlayMode;
 
-  private _loop: boolean;
+  private _loop: boolean | number;
```

Comment:
if `loop` can be either a number, maybe we need to have a validation if it's a valid non negative number.

Reply:
added validation